popup.html

```diff
@@ -2,10 +2,11 @@
   <body>
     <h2>Hide LinkedIn Applicants</h2>
     <p>
-      version: 0.0.7 |
+      version: 0.0.8 |
       <a
         href="https://github.com/garnetred/hide-linkedin-applicants"
         style="color: navy"
+        target="_blank"
```

Comment:
Opens this link in a new tab! I made this mistake in a few extensions so am going back to fix it.